mycore-sword/src/main/java/org/mycore/sword/MCRSwordUtil.java

```diff
@@ -178,7 +176,6 @@ public static MediaResource getZippedDerivateMediaResource(String object) {
     }
 
     private static void addDirectoryToZip(ZipArchiveOutputStream zipOutputStream, Path directory) {
-        MCRSession currentSession = MCRSessionMgr.getCurrentSession();
```

Comment:
`MCRSessionMgr#getCurrentSession()` has a potential side effect. Not sure if removing the method call is okay here.

Reply:
The variable was targeted by UnusedLocalVariable PMD rule as well as greyed out by IntelliJ but if it has negative side effects I can leave it in and surpress the warning. Would that be better?

Reply:
Or just call `MCRSessionMgr.getCurrentSession();` and ignore the return value.

Not sure if there are side effect, but the call could be intentional. That's why I just commented and didn't request a change. Someone with more knowledge should make that decision.